.env.example

```diff
@@ -20,6 +20,7 @@ PRISMA_DATABASE_URL=postgres://postgres:postgrespassword@localhost:5432/postgres
 # Hasura CLI (used with ./hasura/config.yaml)
 HASURA_GRAPHQL_ENDPOINT=http://localhost:8080
 HASURA_GRAPHQL_ADMIN_SECRET=admin-secret
+HASURA_EVENT_SECRET=event-secret
```

Comment:
same as `HASURA_GRAPHQL_ADMIN_SECRET`?

Reply:
it seems more secure to separate them out, since what we are authenticating here is a bit different.